cmake/InstallAndPackage.cmake

```diff
@@ -23,16 +23,26 @@ install(TARGETS openttd
             COMPONENT Runtime
         )
 
-install(DIRECTORY
-                ${CMAKE_BINARY_DIR}/lang
-                ${CMAKE_BINARY_DIR}/baseset
-                ${CMAKE_BINARY_DIR}/ai
-                ${CMAKE_BINARY_DIR}/game
-                ${CMAKE_SOURCE_DIR}/bin/scripts
-        DESTINATION ${DATA_DESTINATION_DIR}
-        COMPONENT language_files
-        REGEX "ai/[^\.]+$" EXCLUDE # Ignore subdirs in ai dir
-)
+if (NOT EMSCRIPTEN)
+    install(DIRECTORY
+                    ${CMAKE_BINARY_DIR}/lang
+                    ${CMAKE_BINARY_DIR}/baseset
+                    ${CMAKE_BINARY_DIR}/ai
+                    ${CMAKE_BINARY_DIR}/game
```

Comment:
It can use AI and GS (from content download), and they might need compatibility scripts.

Reply:
No, those files are already embedded in the `openttd.data` file.